packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -1,10 +1,20 @@
 import {
     LitElement, html, unsafeCSS, TemplateResult, nothing,
 } from 'lit';
-import { defineCustomElement } from '@justeattakeaway/pie-webc-core';
 import { property, state, queryAll } from 'lit/decorators.js';
 import { repeat } from 'lit/directives/repeat.js';
+
+import '@justeattakeaway/pie-button';
```

Comment:
Out of interest – what's the bundle size difference between the before/after of this change? (would be useful to know for when we get back onto the bundling in the next couple of weeks)

Reply:
| Component | Before | After |
| --- | --- | --- |
| `pie-modal` | 30.9kB | 22.7kB |
| `pie-cookie-banner` | 64.7kB |15.7kB |

It's also worth noting that before this PR, the cookie banner bundle also included the modal's dialog polyfill. Now it is only bundled as part of the modal.